pkg/k8sutil/k8sutil.go

```diff
@@ -32,6 +34,19 @@ func GetWatchNamespace() (string, error) {
 	return ns, nil
 }
 
+// getOperatorNamespace returns the namespace the operator should be running in.
+func getOperatorNamespace() (string, error) {
```

Comment:
We should probably export `getOperatorNamespace`, since it would be generally useful.

Also, would you mind removing `func myNS()` from `pkg/leader/leader.go` and replacing any calls to it with `GetOperatorNamespace()`?

Reply:
done